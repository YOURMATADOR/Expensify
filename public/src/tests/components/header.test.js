import {shallow} from 'enzyme';
import Header from '../../components/Header';
import React from 'react'

describe('COMPONETS header', () => {
    test('header', () => {
        const wrapper = shallow(<Header />)
        expect(wrapper).toMatchSnapshot()
    });
});