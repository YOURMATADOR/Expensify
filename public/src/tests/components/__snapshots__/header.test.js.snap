// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONETS header header 1`] = `
<div>
  <h1>
    Titulo chido
  </h1>
</div>
`;
